src/request/node.ts

```diff
@@ -62,6 +64,41 @@ export interface NodeRequestOptions<T> extends RequestOptions {
 	streamData?: boolean;
 	streamEncoding?: string;
 	streamTarget?: WritableStream<T>;
+	redirectOptions?: {
+		limit?: number;
+		count?: number;
+		keepOriginalMethod?: boolean;
+	};
+}
+
+function redirect<T>(resolve: (p?: any) => void, reject: (_?: Error) => void, url: string, options: NodeRequestOptions<T>): boolean {
+	if (!options.redirectOptions) {
+		options.redirectOptions = {};
+	}
+
+	const redirectLimit = options.redirectOptions.limit || defaultRedirectLimit,
+		{ count: redirectCount = 0 } = options.redirectOptions,
+		{ followRedirects = true } = options;
+
+	if (!followRedirects) {
```

Comment:
I don't understand... isn't this already the first thing that happens after we extract `followRedirects`?